be/src/exec/pipeline/scan/connector_scan_operator.cpp

```diff
@@ -302,9 +302,11 @@ ChunkSourcePtr ConnectorScanOperator::create_chunk_source(MorselPtr morsel, int3
         }
     }
 
-    return std::make_shared<ConnectorChunkSource>(this, _chunk_source_profiles[chunk_source_index].get(),
-                                                  std::move(morsel), scan_node, factory->get_chunk_buffer(),
-                                                  _enable_adaptive_io_tasks);
+    // Only use one chunk source profile, so we can see metrics on scan operator level.
+    // Since there is adaptive io tasks feature, chunk sources will be used unevenly,
+    // which leads to sort of "skewed" profile and makes harder to analysis.
+    return std::make_shared<ConnectorChunkSource>(this, _chunk_source_profiles[0].get(), std::move(morsel), scan_node,
+                                                  factory->get_chunk_buffer(), _enable_adaptive_io_tasks);
```

Comment:
The only thing is that the profile of scan is a cumulative time, while other profile means wall time, although cause of parallelism, the original wall time is not accurate.

Reply:
Correct. That's a drawback.  

Before this PR:  time is wall time
After this PR: time is accumulated time of all chunk sources.